### DISCUSSION TITLE
Importing deleted BeatmapSet doesn't update its `Date Added`.

### DISCUSSION THREAD

**Discussion by UserIsntAvailable** [ppy/osu#17399] [Q&A] [answered]
Should be this the expected behavior?  If yes, isn't this counterintuitive?

https://user-images.githubusercontent.com/57047985/159392840-d789ab03-07f7-4572-879c-f2d430a7ab54.mp4

The cause of this is probably over `RealmArchiveModelImporter`. If the BeatmapSet already exists ( `DeletePending = true` ), when re-importing the same map, DeletePending is set to false, but no other changes are made.

Logs if needed:
[logs.zip](https://github.com/ppy/osu/files/8320248/logs.zip)

**Comment by bdach** [comment]
i don't follow. that's how soft delete works. first the map is soft deleted and then restored via reimporting. why would date added change in this case?

**Reply by peppy** [reply]
I would argue this is the general user expectation, as the whole "soft delete" part is hidden to the user.

**Comment by peppy** [comment, marked as answer]
Will be resolved via #17401.
